_build-and-test-all.sh

```diff
@@ -12,18 +12,11 @@ if [ -z "$SPRING_DATA_MONGODB_URI" ] ; then
   echo Set SPRING_DATA_MONGODB_URI $SPRING_DATA_MONGODB_URI
 fi
 
-DOCKER_COMPOSE="docker-compose"
-
-while [ "$1" = "-f" ] ; do
-  shift;
-  DOCKER_COMPOSE="$DOCKER_COMPOSE -f ${1?}"
-  shift
-done
 
 if [ "$1" = "--use-existing" ] ; then
   shift;
 else
-  ${DOCKER_COMPOSE?} down
+  ./gradlew ${database}${mode}ComposeDown
```

Comment:
This `${database}${mode}Compose` is duplicated throughout. Please define in one place with a variable

Reply:
ok